lifecycle_test.go

```diff
@@ -94,6 +97,102 @@ func compareRbArtifacts(t *testing.T, actual services.ReleaseBundleSpecResponse,
 	assert.ElementsMatch(t, actualArtifactsPaths, expected)
 }
 
+func TestReleaseBundleCreationFromMultipleBuildsUsingCommandFlag(t *testing.T) {
+
+	cleanCallback := initLifecycleTest(t, minMultiSourcesArtifactoryVersion)
+	defer cleanCallback()
+	lcManager := getLcServiceManager(t)
+
+	deleteBuilds := uploadBuilds(t)
+	defer deleteBuilds()
+
+	createRbFromMultiSourcesUsingCommandFlags(t, lcManager, GetMultiSourcesForBuilds(), "", tests.LcRbName1, number1, "default", true)
+	defer deleteReleaseBundle(t, lcManager, tests.LcRbName1, number1)
+	assertStatusCompleted(t, lcManager, tests.LcRbName1, number1, "")
+}
+
+func TestReleaseBundleCreationFromMultipleBundlesUsingCommandFlag(t *testing.T) {
+
+	cleanCallback := initLifecycleTest(t, minMultiSourcesArtifactoryVersion)
+	defer cleanCallback()
+	lcManager := getLcServiceManager(t)
+
+	deleteBuilds := uploadBuilds(t)
+	defer deleteBuilds()
+
+	createRbFromSpec(t, tests.LifecycleBuilds12, tests.LcRbName1, number1, true, true)
+	defer deleteReleaseBundle(t, lcManager, tests.LcRbName1, number1)
+
+	createRbFromSpec(t, tests.LifecycleBuilds3, tests.LcRbName2, number2, true, true)
+	defer deleteReleaseBundle(t, lcManager, tests.LcRbName2, number2)
+
+	createRbFromMultiSourcesUsingCommandFlags(t, lcManager, "", GetMultiSourcesForBundles(), tests.LcRbName3, number3, "default", true)
+	defer deleteReleaseBundle(t, lcManager, tests.LcRbName3, number3)
+	assertStatusCompleted(t, lcManager, tests.LcRbName3, number3, "")
+}
+
+func TestReleaseBundleCreationFromMultipleBuildsAndBundlesUsingCommandFlag(t *testing.T) {
+
+	cleanCallback := initLifecycleTest(t, minMultiSourcesArtifactoryVersion)
+	defer cleanCallback()
+	lcManager := getLcServiceManager(t)
+
+	deleteBuilds := uploadBuilds(t)
+	defer deleteBuilds()
+
+	createRbFromSpec(t, tests.LifecycleBuilds12, tests.LcRbName1, number1, true, true)
+	defer deleteReleaseBundle(t, lcManager, tests.LcRbName1, number1)
+
+	createRbFromSpec(t, tests.LifecycleBuilds3, tests.LcRbName2, number2, true, true)
+	defer deleteReleaseBundle(t, lcManager, tests.LcRbName2, number2)
+
+	createRbFromMultiSourcesUsingCommandFlags(t, lcManager, GetMultiSourcesForBuilds(), GetMultiSourcesForBundles(), tests.LcRbName3, number3, "default", true)
+	defer deleteReleaseBundle(t, lcManager, tests.LcRbName3, number3)
+	assertStatusCompleted(t, lcManager, tests.LcRbName3, number3, "")
+}
+
+func TestReleaseBundleCreationFromMultipleSourcesUsingSpec(t *testing.T) {
+
+	cleanCallback := initLifecycleTest(t, minMultiSourcesArtifactoryVersion)
+	defer cleanCallback()
+	lcManager := getLcServiceManager(t)
+
+	deleteBuilds := uploadBuilds(t)
+	defer deleteBuilds()
+
+	createRbFromSpec(t, tests.LifecycleBuilds12, tests.LcRbName1, number1, true, true)
```

Comment:
createRbFromSpec is only creating release bundle from single source, how is it testing multipleResource from a spec?